### DISCUSSION TITLE
Системные требования

### DISCUSSION THREAD

**Discussion by Bossplay228** [skyline-emu/skyline#495] [Q&A] [answered]
У меня вопрос какие нужны системные требование для телефона на этот эмулятор nintendo switch

**Comment by iniMerchant64** [comment]
Sd845 and android 10. Should be fine for light games

**Comment by PixelyIon** [comment, marked as answer]
I should add that these forums are English-only as moderation with multiple languages ends up being fairly difficult. I will be locking the discussion due to this.
